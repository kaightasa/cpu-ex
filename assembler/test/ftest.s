float:
    fadd fr3, fr1, fr2
    fdiv fr6, fr4, fr5
    fmr fr7, fr8