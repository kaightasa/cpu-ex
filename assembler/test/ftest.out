1c611000
24c42800
34e80000
